scripts/us_census/pep/annual_population/preprocess.py

```diff
@@ -0,0 +1,862 @@
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+    data_df = _load_data_df(path=ip_file, file_format="csv", header=1)
+
+    df_cols = [
+        "Year", "Age", "Count_Person", "Count_Person_Male",
+        "Count_Person_Female"
+    ]
+    data_df.columns = df_cols
+    data_df = data_df[(data_df["Age"] == "All Age") &
+                      (data_df["Year"].str.startswith("July"))].reset_index(
+                          drop=True)
+    data_df["Year"] = data_df["Year"].str.replace("July 1, ", "")
+    data_df = data_df.drop(
+        columns=["Age", "Count_Person_Male", "Count_Person_Female"])
+    data_df["Location"] = "country/USA"
+    data_df = data_df[["Year", "Location", "Count_Person"]]
+    return data_df
+
+
+def _process_nationals_2010_2020(ip_file: str, op_file: str) -> None:
+    """
+    Process the nationals data for the year 2010-2020.
+    Args:
+        ip_file (str): Input File Path
+        op_file (str): Output File Path
+    """
+    with open(op_file, 'w', encoding='utf-8') as national_pop_stats:
+        national_pop_stats.write("Year,Location,Count_Person")
+        with open(ip_file, encoding='utf-8') as ipfile:
+            for line in ipfile.readlines():
+                if line.find("POPESTIMATE2010") != -1:
+                    header = line.strip('\n').split(",")
+                elif line.find(USA) != -1:
+                    values = line.strip('\n').split(",")
+        for k, v in dict(zip(header, values)).items():
+            if k.find('POPESTIMATE2') != -1:
+                # k contains values such as POPESTIMATE2010,POPESTIMATE2011,
+                # POPESTIMATE2012 and index at [-4:] provides year value
+                national_pop_stats.write("\n" + k[-4:] + ",country/USA," + v)
+
+
+def _process_nationals_2000_2009(file_path: str) -> pd.DataFrame:
+    """
+    Process the nationals data for the year 2000-2009.
+    Args:
+        file_path (str): Input FIle Path
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+    data_df = pd.DataFrame()
+    data_df = _load_data_df(path=file_path, file_format="csv", header=3)
+
+    pop_cols = [
+        "2000", "2001", "2002", "2003", "2004", "2005", "2006", "2007", "2008",
+        "2009"
+    ]
+    df_cols = ["Region", "042000"] + pop_cols + ["042010", "072010"]
+    data_df.columns = df_cols
+    data_df = data_df[data_df["Region"] == USA]
+    for col in pop_cols:
+        data_df[col] = data_df[col].str.replace(",", "")
+
+    data_df = _unpivot_data_df(data_df, ["Region"], pop_cols)
+    data_df["Location"] = USA_GEO_ID
+    return data_df
+
+
+def _process_states_2000_2009(file_path: str) -> pd.DataFrame:
+    """
+    Process the states data for the year 2000-2009.
+    Args:
+        file_path (str): Input FIle Path
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+    data_df = pd.DataFrame()
+    data_df = _load_data_df(path=file_path, file_format="csv", header=8)
+
+    pop_cols = [
```

Comment:
less error prone to do: `pop_cols = [str(y) for y in range(2000, 2010)]`

Reply:
Done.